CHANGELOG.md

```diff
@@ -1,3 +1,65 @@
+# 0.0.115 - Apr 24, 2023 - "XXX"
+
+## API Updates
+ * The MSRV of the main LDK crates has been increased to 1.48 (#2107).
+ * Attempting to claim an un-expired payment on a channel which has closed no
+   longer fails. The expiry time of payments is exposed via
+   `PaymentClaimable::claim_deadline` (#2148).
+ * `payment_metadata` is now supported in `Invoice` deserialization, sending,
+   and receiving (via a new `RecipientOnionFields` struct) (#2139, #2127).
+ * `Event::PaymentFailed` now exposes a failure reason (#2142).
+ * BOLT12 messages now support stateless generation and validation (#1989).
+ * Max HTLCs in-flight is now configurable via `our_max_accepted_htlcs` (#2138).
+ * `lightning-transaction-sync` feature `esplora-async-https` was added (#2085).
+ * A `ChannelPending` event was added (#2098).
```

Comment:
Can say “used to inform a channel with the given `channel_id` is being opened and pending on-chain confirmation"